apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -218,6 +218,19 @@ export function UpdateProviderPage() {
               />
             )}
           />
+          <Controller
+            control={control}
+            name="name"
+            defaultValue={''}
+            rules={{
+              required: 'Required - Instance name',
+            }}
+            render={({ field }) => (
+              <InputWrapper>
+                <Input {...field} required label="Name" error={errors.name?.message} />
```

Comment:
i think we are missing the fallback for the name, in case the migration is not run yet.

Reply:
Right! Good catch